assets/css/_garage_filter.scss

```diff
@@ -1,37 +1,74 @@
 .m-garage-filter {
   @include font-small;
-  color: #6e717d;
+  font-weight: 400;
+  color: #525560;
   background: $white;
-  border: 1px solid #9fa2ac;
-  border-radius: 0.25rem;
+  border: 1px solid #b3b5bd;
+  border-radius: 6px;
+
+  &:hover {
+    border: 1px solid #b58de2;
+  }
+
+  &:focus-within {
+    border: 1px solid #7c47ae;
+    box-shadow: 0px 0px 0px 2px rgba(170, 78, 242, 0.17);
+  }
 }
 
 .m-garage-filter__header {
-  border-bottom: 1px solid #d3d5d9;
-  padding: 0.5rem;
+  height: 2.25rem;
+  padding: 0 0.5rem 0 1rem;
+  width: 100%;
+  line-height: 1;
   display: flex;
   justify-content: space-between;
+  align-items: center;
+  cursor: pointer;
```

Comment:
**question:** Is this rule still needed now that this class is applied to a `button` instead of a `div`? (Of course, it doesn't hurt anything anyway…)

Reply:
Yep, good catch (and better to avoid extraneous code where possible).